prettytable_test.py

```diff
@@ -636,6 +638,19 @@ def testHtmlOutputFormated(self):
 """.strip()  # noqa: E501
         )
 
+class MarkdownStyleTest(BasicTests):
+    def testMarkdownStyle(self):
+        t = PrettyTable(['Field 1', 'Field 2', 'Field 3'])
+        t.add_row(['value 1', 'value2', 'value3'])
+        t.add_row(['value 4', 'value5', 'value6'])
+        t.add_row(['value 7', 'value8', 'value9'])
+        t.set_style(MARKDOWN)
+        result = t.get_string()
+        assert result.strip() == """| Field 1 | Field 2 | Field 3 |
```

Comment:
Add a newline for readability and consistency with the others
```suggestion
        assert result.strip() == """
| Field 1 | Field 2 | Field 3 |
```